app/ui/browser/actions/main-actions.js

```diff
@@ -11,17 +11,20 @@ specific language governing permissions and limitations under the License.
 */
 
 import * as types from '../constants/action-types';
+import { Page } from '../model';
 import * as profileCommands from '../../../shared/profile-commands';
 import { ipcRenderer } from '../../../shared/electron';
 
 export function createTab(location) {
-  return { type: types.CREATE_TAB, location, instrument: true };
+  const page = location ? new Page({ location }) : new Page();
```

Comment:
Does immutable set undefined? Like, I'd imagine/want `new Page({ location })` to use the default location and not need ternary operator -- if that's not the case, we should probably do something about that, making defaults less useful